compiler/tests-spec/testData/diagnostics/notLinked/dfa/pos/6.fir.kt

```diff
@@ -19,13 +19,13 @@ fun case_1(x: Any?) {
     val y = null
     if (x != y) {
         <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.Any?")!>x<!>
-        <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.Any?")!>x<!>.<!INAPPLICABLE_CANDIDATE!>equals<!>(null)
+        <!DEBUG_INFO_EXPRESSION_TYPE("kotlin.Any?")!>x<!>.<!UNSAFE_CALL!>equals<!>(null)
```

Comment:
Filed https://youtrack.jetbrains.com/issue/KT-44561

There are many more variants in this file as well as: `pos/2.fir.kt` and `pos/7.fir.kt`.